docs/docs/data-structures/tree/node-types.mdx

```diff
@@ -7,27 +7,41 @@ Data on a [SharedTree](./index.mdx) is stored as a [node](./nodes.mdx) of one of
 
 ## Leaf Nodes
 
-A leaf node represents an atomic value, while an interior node represent an object or collection.
+A [leaf](https://fluidframework.com/docs/api/fluid-framework/nodekind-enum#leaf-enummember) node represents an atomic value, while a [TreeNodes](https://fluidframework.com/docs/api/fluid-framework/treenode-class) represents an object or collection.
 The following leaf node types are available:
 
--   **boolean**: Works identically to a JavaScript boolean.
--   **number**: Works identically to a JavaScript JavaScript number.
--   **string**: Works identically to a JavaScript string.
--   **null**: Works identically to a JavaScript null.
--   **FluidHandle**: A handle to a Fluid DDS or Data Object in the current container. For more information about handles see [Handles](../../concepts/handles.mdx).
+-   [**boolean**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#boolean-property): Works identically to a JavaScript boolean.
+-   [**number**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#number-property): Works identically to a JavaScript JavaScript number except does not support -0, NaN or Infinity.
+-   [**string**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#string-property): Works identically to a JavaScript string except for not supporting unpaired UTF-16 surrogate pairs.
+-   [**null**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#null-property): Works identically to a JavaScript null.
+-   [**FluidHandle**](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#handle-property): A handle to a Fluid DDS or Data Object in the current container. For more information about handles see [Handles](../../concepts/handles.mdx).
 
-The following subsections describe the available internal node types.
+## TreeNodes
```

Comment:
nit: TreeNode?

Reply:
This matches "Leaf Nodes" above which is plural, and the ones below which are also plural.